### PR TITLE
Reinstate lost README file

### PR DESCRIPTION
**Pull request by barrie-cooper**
The READEME.md file was deleted by accident in June, now reinstating

### CODE REVIEW

**Comment by barrie-cooper** [comment]
@rockydevnet This is the README file that last existed in this repo but I'm guessing that everything after line 13 should be removed?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+*Note that this section is draft, based on what we are doing for the [File Share Service (FSS)](https://github.com/UKHO/file-share-service/)*
+
+### Using ESS in Postman
+
+To generate an authorization token in Postman follow these steps:
+
+#### Get Authorization for the environment's App Registration
+
+Ask ### TBC ### (or service Owner if service is live) to give you Roles for the ESS App Registration:
```
Comment (rockydevnet) [inline README.md:27]:
```suggestion
Ask Calypso (or service Owner) to give you Roles for the ESS App Registration:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Fill in the OAuth section, click get new access token, login from the popup window
+
+Grant Type - Implicit
+
+Auth URL - `https://login.microsoftonline.com/9134ca48-663d-4a05-968a-31a42f0aed3e/oauth2/v2.0/authorize`
+
+Client ID:
+
+* DEV ### TBC ###
```
Comment (rockydevnet) [inline README.md:47]: Could you add the relevant Client IDs here please?

```suggestion
| Environment | Client ID |
| -- | -- |
| DEV | 80a6c68b-59aa-49a4-939a-7968ff79d676 |
| QA | 644e4406-4e92-4e5d-bdc5-3b233884f900 |
| LIVE | fd4cb7ab-ed8f-4398-bafa-4f83284febbe |

File: README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* QA ### TBC ###
+* LIVE ### TBC ###
+
+Scope:
+
+* DEV ### TBC ###/.default
```
Comment (rockydevnet) [inline README.md:53]:
```suggestion
| Environment | Scope |
| -- | -- |
| DEV | 80a6c68b-59aa-49a4-939a-7968ff79d676/.default |
| QA | 644e4406-4e92-4e5d-bdc5-3b233884f900/.default |
| LIVE | fd4cb7ab-ed8f-4398-bafa-4f83284febbe/.default |

**Review by rockydevnet — APPROVED** [review]
LGTM, I've added the details for the Client IDs and Scope we didn't have back in the summer, but otherwise everything in here should be.

**Pull request merged**